Buildings/Fluid/ZoneEquipment/FanCoilUnit/package.order

```diff
@@ -0,0 +1,4 @@
+FanCoilUnit
```

Comment:
alphabetical order and the "FanCoilUnit" seems to be duplicated as the package is also "FanCoilUnit"